### DISCUSSION TITLE
Support MERGE operation

### DISCUSSION THREAD

**Discussion by Rid** [supabase/supabase#18463] [Feature Requests]
Postgres 15 introduced the merge operation: https://supabase.com/blog/new-in-postgres-15

It would be great if supabase-js can support this operation to simplify upserts.

**Comment by steve-chavez** [comment]
You can do the equivalent of MERGE with `upsert()`, internally it uses ON CONFLICT - which should give you similar capabilities.

See: https://pganalyze.com/blog/5mins-postgres-15-merge-vs-insert-on-conflict
